cgi-bin/sitemap.js

```diff
@@ -84,12 +84,12 @@ async function run(params) {
   ]);
 
   if (queryYAML.status !== 200) {
-    log.error(`unable to fetch helix-query.yaml: ${queryYAML.status}`);
```

Comment:
well... I would differentiate between 404 and 5xx errors....

Reply:
Makes sense. Just raising the log level or returning another body?

Reply:
Hmm, if the `status` isn't either 200 or 404, an error is thrown anyway, and the runtime action is exited abnormally 🙄